src/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant/ClientCredentialsConstants.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace GSoft.Extensions.Http.Authentication.ClientCredentialsGrant;
+
+public static class ClientCredentialsConstants
+{
+    public const string BackchannelHttpClientName = "GSoft.Authentication.ClientCredentialsGrant";
```

Comment:
This internal HTTP client name is exposed for consumers to remove or replace the built-in retry policy we've put in place.